app/models/publisher.rb

```diff
@@ -198,7 +203,30 @@ def has_verified_channel?
     channels.any?(&:verified?)
   end
 
+  def last_status_update
+    status_updates.first
+  end
+
   private
+  
+  def set_created_status
+    created_publisher_status_update = PublisherStatusUpdate.new(publisher: self, status: "created")
```

Comment:
Would you convert the statuses into constants? I recommend keep track of statuses in something similar to:

`STATUSES = [ONBOARDING, CREATED, ACTIVE, SUSPENDED]`

Reply:
I added a list a constant `ALL_STATUSES` on the `PublisherStatusUpdates` model, and added a validation that the status must be in the list of all statuses.  Please let me know if you had something else in mind @yachtcaptain23.